releasenotes/notes/enable-data-stream-monitoring-for-kinesis-d5685ec1ac504cc3.yaml

```
@@ -0,0 +1,9 @@
+---
+prelude: >
+  Data Streams Monitoring (DSM) has added support for AWS Kinesis
+
+features:
+  - |
+     DSM: Adds DSM support for AWS Kinesis.  For information about DSM, see the [official documentation](https://docs.datadoghq.com/data_streams/). 
```

something like this should work

```suggestion
     DSM: Adds DSM support for AWS Kinesis.  For information about DSM, see the `official documentation <https://docs.datadoghq.com/data_streams/>`_ . 
```